### PR TITLE
[zh] Update text for ist0119 - ist0130

### PR DESCRIPTION
**Pull request by windsonsea**
Updated zh text for consistency in. this folder.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/reference/config/analysis/ist0119/index.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当认证策略指定使用 JWT 认证但目标 [Kubernetes 服务](https://kubernetes.io/zh-cn/docs/concepts/services-networking/service/) 配置不正确时，会出现此消息。
-正确定位到 Kubernetes 服务需要使用 http|http2|https 前缀来命名端口
-（请参见 [协议选择](/zh/docs/ops/configuration/traffic-management/protocol-selection/)），并且还需要协议使用 TCP；协议留空也可以，因为其默认值就是 TCP。
+当认证策略指定使用 JWT 认证但目标
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0119/index.md:8]: I actually haven't seen this message before, and have no idea about what this meesage is doing, I'm going to check it.

Comment (hanxiaop) [reply content/zh/docs/reference/config/analysis/ist0119/index.md:8]: I think the analyzer that generates the message has been removed for a long time, see https://github.com/istio/istio/pull/22706. I think we can just retrie this message and remove the docs from the istio.io. Would you like to do this?

Comment (hanxiaop) [reply content/zh/docs/reference/config/analysis/ist0119/index.md:8]: See https://github.com/istio/istio/pull/28830/files#diff-5ab445fd2493bb54c4667c7e74f7c0e5773ab35bf24fc5e639ab24d90a333274R216 as an example, and you can modify https://github.com/istio/istio/blob/master/pkg/config/analysis/msg/messages.yaml#L179-L192 in the same way.

Comment (hanxiaop) [reply content/zh/docs/reference/config/analysis/ist0119/index.md:8]: After the message is cleaned up, the file will be synced to https://github.com/istio/istio.io/blob/master/data/analysis.yaml, then you can delete the corresponding message description in istio.io

**Review by hanxiaop — COMMENTED** [review]
Others lgtm

File: content/zh/docs/reference/config/analysis/ist0119/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0119/index.md:8]: Well received. Will update the istio repo soon before remove ist0119 here. Thanks for your guidance 👍

**Comment by hanxiaop** [comment]
@windsonsea Why close the PR? I think other modifications look good?

**Comment by windsonsea** [comment]
Oops, accidentally closed...

**Pull request closed without merging**